src/object/srv_internal.h

```diff
@@ -90,6 +91,11 @@ struct migrate_pool_tls {
 	ABT_mutex		mpt_inflight_mutex;
 	int			mpt_inflight_max_ult;
 	uint32_t		mpt_opc;
+
+	/* This is only used for upgrade process for the new layout
+	 * version to be upgraded to
+	 */
+	uint32_t		mpt_new_layout_ver;
```

Comment:
minor, looks better to put together with mpt_global_version/mpt_layout_version and explain a little bit about the difference. thx